keyboards/rominronin/katana60/rev2/keymaps/via/keymap.c

```diff
@@ -0,0 +1,71 @@
+/* Copyright 2019 rominronin
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// MacOS based definitions.
+#define K_SPCFN LT(SYMB, KC_SPACE) // Tap for space, hold for symbols layer
+#define K_PRVWD LALT(KC_LEFT)      // Previous word
+#define K_NXTWD LALT(KC_RIGHT)     // Next word
+#define K_LSTRT LGUI(KC_LEFT)      // Start of line
+#define K_LEND  LGUI(KC_RIGHT)     // End of line
+#define UNDO    LGUI(KC_Z)         // UNDO
+#define CUT     LGUI(KC_X)         // CUT
+#define COPY    LGUI(KC_C)         // COPY
+#define PASTE   LGUI(KC_V)         // PASTE
```

Comment:
```suggestion
```
I'm not sure these make too much sense, especially since they are focused on Mac.
I'd rather see "standard" keycodes in the keymap, since everybody will tune their layout anyway.

Reply:
Changed a few of them to the generic counterpart (KC_HOME/KC_END) and took the others out completely.